### DISCUSSION TITLE
Multiple or like conditions

### DISCUSSION THREAD

**Discussion by NotDemin** [ujjwalguptaofficial/JsStore#371] [Q&A]
Hello, im struggling to understand the query syntax for my sql query.

My query in SQL is something like :

`SELECT * FROM table WHERE table.text LIKE '%foo%' AND (table.text LIKE '%bar%' OR table.text LIKE '%foobar%' OR ...) AND ...`

I can have as many or conditions as I want. How should I write this in JSstore ?

Thanks

**Comment by ujjwalguptaofficial** [comment]
Use Array query for more complex queries. Have a look at this - 

Here is an example - 

## SQL

```
Select * from Products where supplierId = 1 and (categoryId = 1 and price = 18) or(categoryId = 2 and price = 39)
```

## JsStore

```
select({
    from: "Products",
    where: [{
        supplierId: 1,
    }, {
        categoryId: 1,
        price: 18,
    }, {
        or: {
            categoryId: 2,
            price: 39,
        }
    }]
});
```

Take a look at this link - https://jsstore.net/docs/where/or#more-examples
